source/collaborate/schedule-messages.rst

```diff
@@ -0,0 +1,43 @@
+Schedule messages
+==================
+
+.. include:: ../_static/badges/ent-pro-cloud-selfhosted.rst
+  :start-after: :nosearch:
+
+There are times when you want to send a message, but you don't want it sent immediately. For example, it may be after working hours for the recipient. 
+From Mattermost v10.3, using Mattermost in a web browser or the desktop app, you can schedule messages to send in the future. 
+
+Compose a message, then select the right side of the **Send** icon |send-icon| to schedule when the message will be sent. You can choose from a fixed time or set a custom time.
+
+.. image:: ../images/scheduled-messages.gif
+    :alt: Compose a message and select the right side of the Send icon to schedule when the message will be sent.
+
+.. tip::
+
+    When scheduling a direct message, Mattermost displays both your local time and the recipient's local time.
+
+Manage scheduled messages
+-------------------------
+
+Once you schedule a message to send in the future, that message is available within the **Drafts** view, under the **Scheduled** tab.
+
+You can manage scheduled messages with the following actions:
+
+- **Delete scheduled post**: Confirm that you want to delete the message.
+- **Edit scheduled post**: Make changes to the draft inline before its sent.
+- **Copy text**: Copy the draft text.
+- **Reschedule message**: Change when the message should be sent.
+- **Send now**: Confirm that you want to send the message immediately.
+
+.. image:: ../images/manage-scheduled-message.png
+    :alt: Once a message is scheduled, you can delete, edit, and reschedule it, copy the message text, or send it immediately.
+
+Troubleshoot scheduled messages
+-------------------------------
+
+If a scheduled message fails to send, you'll be alerted with a banner at the top of the Mattermost screen. Scheduled messages may fail for the following reasons:
+
+- You're no longer a member of the channel.
+- The channel is archived.
+- The channel is read-only.
+- The Mattermost server didn't respond. Mattermost will try to send the message for up to 30 minutes.
```

Comment:
This point isn't completely right. I propose updating this to something like "Mattermost server was down for more than 24 hours". We process scheduled posts up to 24 hours old, so if the server is down for a couple of hours, its fine, but more than 24 hours will cause some scheduled posts to be skipped.